apps/arweave/src/ar_coordination.erl

```diff
@@ -355,14 +396,15 @@ send_h2(Peer, Candidate) ->
 
 add_mining_peer({Peer, StorageModules}, State) ->
 	Partitions = lists:map(
-		fun({PartitionId, _PartitionSize, _PackingAddr}) -> PartitionId end, StorageModules),
+		fun({PartitionID, _PartitionSize, PackingAddr}) ->
+			{PartitionID, PackingAddr} end, StorageModules),
 	?LOG_INFO([{event, cm_peer_updated},
 		{peer, ar_util:format_peer(Peer)}, {partitions, io_lib:format("~p", [Partitions])}]),
```

Comment:
I think this log might need to be updated to encode the `PackingAddr` that's been added to the `Partitions` list